src/Components/OATGraphViewer/Internal/OATGraphCustomEdge.tsx

```diff
@@ -49,35 +49,127 @@ const OATGraphCustomEdge: React.FC<IOATGraphCustomEdgeProps> = ({
     } = useContext(ElementsContext);
     const graphViewerStyles = getGraphViewerStyles();
     const theme = useTheme();
+    sourceY = sourceY - 6;
```

Comment:
Why `6`? Let's define it as a well known constant.

Also, I'd prefer not to edit props. We should create a new variable.